src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java

```diff
@@ -298,30 +352,32 @@ public FormValidation doCheckNameOrUrl(@QueryParameter final String value) {
         }
 
         public FormValidation doCheckUuid(@QueryParameter final String value) {
-            if (StringUtils.isBlank(value)) {
```

Comment:
level of nesting is growing here, unnecessarily, and this degrades readability.
Unless there is a logic missed with "linearization", why doing it ?